File: Rust/fizzbuzz.rs
// FizzBuzz implementation in Rust Programming Language
// Author: @QaidVoid

fn fizzbuzz() {
    for n in 1..=100 {
        let val = match (n % 3, n % 5) {
            (0, 0) => "FizzBuzz".into(),
            (0, _) => "Fizz".into(),
            (_, 0) => "Buzz".into(),
            _ => n.to_string()
        };

        if n == 100 {
            print!("{}", val);
        } else {
            print!("{}, ", val);
        }
    }
}

fn main() {
    fizzbuzz();
}

File: Perl/fizzbuzz.pl
#// $fizzbuzz Perl
#// Author: @rhopkins7428

my @count = ( 1 .. 100 );

foreach my $fizzbuzz ( @count )
{

    if ( $fizzbuzz % 3 == 0 )
    {
        print "fizz";
    }

    if ( $fizzbuzz % 5 == 0 )
    {
        print "buzz";
    }

    if ( $fizzbuzz % 3 != 0 and $fizzbuzz % 5 != 0 )
    {
        print $fizzbuzz;
    }

    print "\n";

}

